### DISCUSSION TITLE
CopyQ : Show what's in clipboard memory

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [hluk/CopyQ#1785] [Ideas]
hi
Too often i paste the wrong because i read only the top line in CopyQ but there is something other in it's memory which i can't easy see alas.
Perhaps it would help if one could optionally activate the memory line on top of all columns?
With another background color it would be quick and easy to see what's actually in clipboard memory :)
thx

**Comment by hluk** [comment]
Ctrl+Shift+C from main application window shows current clipboard content.
